addons/context_actions/CfgContext.hpp

```diff
@@ -315,6 +315,12 @@ class EGVAR(context_menu,actions) {
             statement = QUOTE(_objects call FUNC(refuelVehicles));
             icon = "\a3\ui_f\data\igui\cfg\simpleTasks\types\refuel_ca.paa";
         };
+        class UnloadViV {
+            displayName = "$STR_A3_ModuleDepot_Unload";
+            condition = QUOTE(!(getVehicleCargo _hoveredEntity isEqualTo []) || {!isNull isVehicleCargo _hoveredEntity});
+            statement = QUOTE(if (isNull isVehicleCargo _hoveredEntity) then {_hoveredEntity setVehicleCargo objNull} else {objNull setVehicleCargo _hoveredEntity});
```

Comment:
Move code to functions like action above.

Reply:
6d79572